### PR TITLE
Add GHC 9.2 support

### PR DESCRIPTION
**Pull request by brandon-leapyear**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -31,6 +31,8 @@ matrix:
     addons: {apt: {packages: [libgmp-dev]}}
   - env: BUILD=stack STACK='stack --stack-yaml=stack-9.0.yaml'
     addons: {apt: {packages: [libgmp-dev]}}
+  - env: BUILD=stack STACK='stack --stack-yaml=stack-9.2.yaml'
```
Comment (blackgnezdo) [inline .travis.yml:34]: We switched to github actions for CI. Please update  `.github/workflows/stack-ci.yml`.

Could you do me favor and delete `.travis.yml`? It only confuses us...

**Review by blackgnezdo — CHANGES_REQUESTED** [review]

File: .travis.yml (hunk 1 above)
Comment (brandon-leapyear) [reply .travis.yml]: Seems like I didn't need to change anything in `stack-ci.yml`? Just `matrix.yaml`? (Not sure why matrix is defined in another file; why not just define as a normal matrix in the github actions config?)

**Review by blackgnezdo — APPROVED** [review]

**Comment by blackgnezdo** [comment]
Thanks @brandon-leapyear!

**Comment by brandon-leapyear** [comment]
@blackgnezdo when do you think you'll get around to making a release with 9.2 support?

**Comment by blackgnezdo** [comment]
You are right, it's about time. I'm retesting with ghc-9.2.2 now and should be able to publish this week.

**Comment by brandon-leapyear** [comment]
Thanks!

**Pull request merged**